README.md

```diff
@@ -300,88 +300,129 @@ ruff path/to/code/ --select F401 --select F403
 
 See `ruff --help` for more:
 
+<!-- Begin auto-generated cli help. -->
 ```shell
 Ruff: An extremely fast Python linter.
 
 Usage: ruff [OPTIONS] [FILES]...
 
 Arguments:
   [FILES]...
+          
 
 Options:
       --config <CONFIG>
           Path to the `pyproject.toml` or `ruff.toml` file to use for configuration
+
```

Comment:
Any idea where all the newlines are coming from?

Reply:
Good point, I'll go looking.

Reply:
Just got back from my deep-dive into clap.

In the end, I just had to go from `render_long_help` to `render_help`. :sweat_smile:

There is some trailing whitespace after the `FILES` lines, but I don't think that's an issue.

Reply:
All good. Thanks again :)

Reply:
@charliermarsh I've noticed that the trailing whitespace seems to cause issues. In https://github.com/charliermarsh/ruff/commit/74903f23d69172474116ec3e05d182cdf9179415 it got removed, which fails CI again. Shall we adjust the cmd to strip trailing whitespace from the generated output?

Reply:
Hmm yeah, I guess so. I wonder how it got removed? Is it machine-specific? Or did my editor do it automatically, or something?

Reply:
Are you working on this? I could do it later if needed.

Reply:
I got it :)